### ISSUE TITLE
add SDXL support to DPM Solver++

### ISSUE THREAD

**Issue by czkoko** [GuernikaCore/Schedulers#1]
hello @GuiyeC 
The apple/ml-stable-diffusion project has added SDXL support to DPMSolverMultistepScheduler some time ago.
[https://github.com/apple/ml-stable-diffusion/commit/94814cfa41935efd8151a43758360a54e4e3c5d5](https://github.com/apple/ml-stable-diffusion/commit/94814cfa41935efd8151a43758360a54e4e3c5d5)

At present, only EulerA is compatible with SDXL in Guernika, and other schedulers have different degrees of image quality problems.
Hope you can fix the problem of DPM Solver++ for SDXL.

**Comment by czkoko** [comment]
hello @GuiyeC
SDXL Turbo seems to need DPM++ SDE Karras to present perfect picture quality, and other schedulers will have image quality problems like SDXL LCM.
[TurboVisionXL](https://civitai.com/models/215418) can work with DPM++ SDE Karras to achieve very good image quality and higher resolution.

**Comment by GuiyeC** [comment]
@czkoko I have pushed quite a lot of updates to this repository, including DPM Solver ++, I'm not sure why you said it wasn't compatible with SDXL but I added the Karras option.
Also why do you say SDXL Turbo needs DPM++ SDE Karras? where did you see that? it uses EulerA by default.

**Comment by czkoko** [comment]
@GuiyeC
Because I see that all the developers of the fine-tuned model recommend using DPM++ SDE Karras. They have been tested for a long time. This scheduler can indeed provide better details for SDXL.

At the same time, I'm sorry that I just started to use EulerA to test SDXL Turbo within 5 steps, so I feel that its picture quality is poor. I just found that EulerA needs at least 10 steps to have perfect image quality.

Thank you for so many days of hard work to add the common scheduler. 
Looking forward to Guernika V8.
